src/content/learn/editor-setup.md

```diff
@@ -5,58 +5,61 @@ title: Editor Setup
 
 ### Linting {/*linting*/}
 
-Code linters find problems in your code as you write, helping you fix them early. [ESLint](https://eslint.org/) is a popular, open source linter for JavaScript. 
+Kod linter araçları kodunuzu yazarken problemleri bulmanızı ve düzeltmenizi sağlar. [ESLint](https://eslint.org/) popüler açık kaynak kodu Javascript linter.
 
-* [Install ESLint with the recommended configuration for React](https://www.npmjs.com/package/eslint-config-react-app) (be sure you have [Node installed!](https://nodejs.org/en/download/current/))
-* [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+* [React için önerilen ESLint ayarları](https://www.npmjs.com/package/eslint-config-react-app) ([Node]'un yüklü olduğundan emin olun!(https://nodejs.org/en/download/current/))
+* [ESLint'i, VSCode'a resmi eklenti ile entegre edin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 
-**Make sure that you've enabled all the [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) rules for your project.** They are essential and catch the most severe bugs early. The recommended [`eslint-config-react-app`](https://www.npmjs.com/package/eslint-config-react-app) preset already includes them.
+** [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) kurallarını projeniz için etkinleştirdiğinizden emin olun.
+** Bu özellikler önemlidir ve en ciddi hataları erken aşamada yakalarlar. Önerilen [`eslint-config-react-app`](https://www.npmjs.com/package/eslint-config-react-app) ön ayar bunları içeriyor.
 
-### Formatting {/*formatting*/}
+### Biçimlendirme {/*formatting*/}
 
-The last thing you want to do when sharing your code with another contributor is get into an discussion about [tabs vs spaces](https://www.google.com/search?q=tabs+vs+spaces)! Fortunately, [Prettier](https://prettier.io/) will clean up your code by reformatting it to conform to preset, configurable rules. Run Prettier, and all your tabs will be converted to spaces—and your indentation, quotes, etc will also all be changed to conform to the configuration. In the ideal setup, Prettier will run when you save your file, quickly making these edits for you.
+Kodunuza katkıda bulunan başka bir kişiyle yapmak isteyeceğiniz son şey [sekmeler ve boşluklar](https://www.google.com/search?q=tabs+vs+spaces) hakkında bir tartışmaya girmektir! Neyse ki, [Prettier](https://prettier.io/) kodunuzu önceden ayarladığınız kurallara uyacak şekilde biçimlendirerek düzenler ve temizler. Prettier'ı çalıştırın ve tüm sekmeleriniz boşluklara dönüştürülecek ve girintileriniz, alıntılarınız vb. de yapılandırmaya uyacak şekilde değiştirilecektir. İdeal kurulumda Prettier, dosyanızı kaydettiğinizde çalışacak ve bu düzenlemeleri sizin için hızlıca yapacaktır.
```

Comment:
Couldn't decide it either. In microsoft office, they translate it "sekme"

Reply:
Yes, it's a tricky one. I think it's more like "çıkıntı" rather than "sekme". My suggestion is going with tab since people can be confused about this. I even thought editor tabs instead of indentation tabs 😅

Reply:
Yeah, I saw a translation with "Girinti durağı", ok I am changing it to "tab"